common/types/shared.go

```diff
@@ -2731,7 +2731,14 @@ func (v *ActiveClusters) DeepCopy() *ActiveClusters {
 	if v.AttributeScopes != nil {
 		result.AttributeScopes = make(map[string]ClusterAttributeScope, len(v.AttributeScopes))
 		for scopeType, scope := range v.AttributeScopes {
-			result.AttributeScopes[scopeType] = scope
+			copiedScope := ClusterAttributeScope{}
```

Comment:
separate PR: https://github.com/cadence-workflow/cadence/pull/7394 will be rebase in